packages/base/card-api.gts

```diff
@@ -129,9 +130,9 @@ function isStaleValue(value: any): value is StaleValue {
     return false;
   }
 }
-const deserializedData = new WeakMap<Card, Map<string, any>>();
```

Comment:
This should probably use Card type since it’s referring to a map of fields that only composite cards contain